src/BenchmarkDotNet/Extensions/StringAndTextExtensions.cs

```diff
@@ -37,6 +37,16 @@ internal static string EscapeSpecialCharacters(this string str, bool quote)
             return Microsoft.CodeAnalysis.CSharp.SymbolDisplay.FormatLiteral(str, quote);
         }
 
+        /// <summary>
+        /// Escapes UNICODE control character
+        /// </summary>
+        /// <param name="c">char to escape</param>
+        /// <param name="quote">True to put (single) quotes around the character literal.</param>
+        internal static string EscapeSpecialCharacter(this char c, bool quote)
+        {
+            return Microsoft.CodeAnalysis.CSharp.SymbolDisplay.FormatLiteral(c, quote);
```

Comment:
it's really great idea to use C# compiler itself for escaping code used in a C# source file 👍